Explorer/Assets/DCL/SceneLoadingScreens/SceneLoadingScreenController.cs

```diff
@@ -185,7 +185,8 @@ private void ShowTip(int index)
 
             viewInstance.HideAllTips();
             viewInstance.ShowTip(index);
-            viewInstance.Background.color = tips.Tips[index].BackgroundColor;
+
+            viewInstance.Background.material.SetColor("_Color1", tips.Tips[index].BackgroundColor);
```

Comment:
This should be a responsibility of the view. You can encapsulate the complexity into a method: `viewInstance.SetBackgroundColor(tips.Tips[index].BackgroundColor)`

Reply:
Changed